core/src/trezor/lvglui/scrs/template.py

```
@@ -881,6 +881,189 @@ def on_click(self, event_obj):
+                self.show_full_data.add_style(
+                    StyleWrapper().text_font(font_GeistSemiBold26).pad_hor(24), 0
+                )
+                self.show_full_data.align(lv.ALIGN.CENTER, 0, 0)
+                self.show_full_data.remove_style(None, lv.PART.MAIN | lv.STATE.PRESSED)
+                self.show_full_data.add_event_cb(self.on_click, lv.EVENT.CLICKED, None)
+
+    def on_click(self, event_obj):
+        code = event_obj.code
+        target = event_obj.get_target()
+        if code == lv.EVENT.CLICKED:
+            if target == self.show_full_data:
```

Ruff (0.8.2)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)